server/utils/db/db_utils.py

```
@@ -3023,6 +3023,31 @@ def initialize_tables():
+                    ALTER TABLE users ADD COLUMN IF NOT EXISTS email_verification_code VARCHAR(64);
+                    ALTER TABLE users ADD COLUMN IF NOT EXISTS email_verification_code_expires_at TIMESTAMP;
+                    ALTER TABLE users ADD COLUMN IF NOT EXISTS email_verification_attempts INTEGER DEFAULT 0;
+                """)
+
+                if not column_existed:
+                    cursor.execute("UPDATE users SET email_verified = TRUE;")
+                    logging.info("Backfilled email_verified=TRUE for existing users.")
+
+                conn.commit()
+                logging.info("Ensured email verification columns exist on users table.")
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)